doc/py_tutorials/py_feature2d/py_brief/py_brief.markdown

```diff
@@ -43,6 +43,10 @@ points than for SURF points.
 In short, BRIEF is a faster method feature descriptor calculation and matching. It also provides
 high recognition rate unless there is large in-plane rotation.
 
+STAR in OpenCV 
+------
+STAR is a feature detecter dervided from CenSurE. Unline CenSurE however, which uses polygons like squares, hexagons and octagons to approach a circle, Star emulates a circle with 2 overlapping squares: 1 upright and 1 45-degree rotated. These polygons are bi-level. The can be seen as polygons with thick borders. The borders and the enclosed area have weights of opposing signs. This has better computational characteristics than other scale-space detectors and it is capable of real-time implementation. In contrast to SIFT and SURF, which find extrema at subsampled pixels that compromises accuracy at larger scales, CenSurE creates a feature vector using full spatial resolution at all scales in the pyramid. 
```

Comment:
Please fix typos.

Avoid very long lines. Consider starting new sentences on new line. This line breaks don't start new paragraph.

Remove trailing white spaces (check build logs).